application/core/ASPA_Controller.php

```diff
@@ -16,13 +16,28 @@ class ASPA_Controller extends CI_Controller
      * @var array All the information for this event (retrieved from google sheet).
      */
     protected $eventData;
-
+    protected $org;
     function __construct()
     {
         parent::__construct();
         // $this->load->helper();
 		// $this->load->model();
         $this->eventData = $this->loadEventData();
+        $this->org = $this->loadOrganisationData();
+    }
+
+    /**
+     * This function handles the loading the Organisation model
+     *
+     * @return array
+     */
+    public function loadOrganisationData(){
```

Comment:
the function should be styled

```
public function loadOrganisationData()
{
     // code here
}
```

To match PEP style guides, if you can also change `loadEventData()` to match as well that'll be awesome 👍